packages/devtools/test/composite.test.ts

```diff
@@ -555,13 +554,13 @@ describe('composite', () => {
       })
 
       test('with indices', async () => {
-        const postComposite = await Composite.create({ ceramic, schema: postSchema })
-        const ratingComposite = await Composite.create({ ceramic, schema: ratingSchema })
-        const noteComposite = await Composite.create({ ceramic, schema: noteSchema })
+        const [postComposite, ratingComposite, noteComposite] = await Promise.all([
+          Composite.create({ ceramic, schema: postSchema }),
+          Composite.create({ ceramic, schema: ratingSchema }),
+          Composite.create({ ceramic, schema: noteSchema }),
+        ])
         const mergedComposite = postComposite.merge([ratingComposite, noteComposite]).toJSON()
-        expect(mergedComposite).toHaveProperty('indices.Post')
-        expect(mergedComposite).toHaveProperty('indices.Rating')
-        expect(mergedComposite).toHaveProperty('indices.Note')
+        expect(Object.keys(mergedComposite.indices ?? {})).toHaveLength(4)
```

Comment:
This looks to originally have 3 indices, as checking exact property. Is 4 correct?

Reply:
Yes, the `postSchema` now also contains the `Comment` model creation that defines one index.

Reply:
Can we add back at least one check for the index matching, e.g. indices should contain Post.

Reply:
OK, I replaced the length check by a snapshot